python/runfiles/BUILD.bazel

```diff
@@ -40,12 +48,27 @@ py_wheel(
         "Development Status :: 5 - Production/Stable",
         "License :: OSI Approved :: Apache Software License",
     ],
-    description_file = "README.md",
+    description_file = "README.rst",
```

Comment:
note, this change is actually required, otherwise pypi rejects the package. There's some way to configure the package metadata to allow markdown instead, but it seems easier to just use their defaults.